### PR TITLE
Add unlock steps for legacy android devices

### PR DESCRIPTION
**Pull request by NeoTheThird**
Since the UBports Installer supports these starting with the next version, it's time to get these in there. Does not break compatibility with older versions of the installer.

### CODE REVIEW

**Comment by NeoTheThird** [comment]
Added model confirmation prompts, thanks @wayneoutthere for the good idea :)

**Comment by wayneoutthere** [comment]
I don't remember this idea.  But i'm glad it helped! :-0

File: v1/flo.json [hunk 1]
```diff
@@ -14,6 +14,10 @@
       "description": "With the device powered off, hold Volume Down + Power until the bootloader appears.",
       "image": "phone_power_down",
       "button": true
+    },
```
Comment (Flohack74) [inline v1/flo.json]: You are missing a confirm_model entry here probably

Comment (NeoTheThird) [reply v1/flo.json]: yep, thanks

**Pull request merged**